driver-benchmarks/src/main/com/mongodb/benchmark/jmh/codec/BsonUtils.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2016-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.mongodb.benchmark.jmh.codec;
+
+import org.bson.BsonBinaryWriter;
+import org.bson.BsonDocument;
+import org.bson.codecs.Codec;
```

Comment:
```suggestion
import ogg.bson.codecs.BsonDocumentCodec;
import org.bson.codecs.Codec;
```

New-up BsonDocumentCodec directly.